### ISSUE TITLE
Staging Review finding: Unnecessary screen reader text on some icons

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#82713]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Representation Management

**Product:** Appoint a Representative MVP + Representative Status Widget MVP
**Feature:** Appoint a Representative MVP + Representative Status Widget MVP

## Findings details
**VA.gov Experience Standard - issue:** [User encounters unnecessary repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When a user has a current accredited representative, the card displaying that information has several icons. Some of those icons communicate information to users in a way not otherwise represented in text, some don't (making them decorative). All of them have a text alternative for screen reader users, which is great for two of them but can lead to some unnecessary repetition for the decorative ones.

The two that are decorative and are resulting in unnecessary repetition:
- The `account_circle` icon is announced as "Your representative," followed by the text "Your current accredited representative"
- The `file_download` icon is announced as "Download your accredited representative's contact information," followed by a link with text "Download your accredited representative's contact information"
- The `search` icon is announced as "Learn about accredited representatives," followed by a link with text "Learn about accredited representatives"

The other two icons (`location_on` and `phone`) are announced with phrases that describe the links that follow, which is useful information.

<details><summary>Link, screenshot or steps to recreate</summary>

![Card with heading "Your current accredited representative" and information about American Legion. The heading, address, phone number, and a learn more link are each visually indicated with an icon.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/71535-Brian-01.png?raw=true)

![Card with heading "Your current accredited representative" and information about Katrina Eagle. The heading, address, phone number, a download link, and a learn more link are each visually indicated with an icon.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/71535-Brian-02.png?raw=true)

</details>

## Recommended action

On the `account_circle`, `file_download`, and `search` icons, remove the `srtext` prop from the component.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.1.1 Non-text Content - Level A](https://www.w3.org/TR/WCAG21/#non-text-content)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by oddball-lindsay** [comment]
@shiragoodman @briandeconinck I'm getting a "Not Found" when I try to visit these links - do you mind taking a look?
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/142453186/fed0bd24-8cd6-4c80-8a89-13196888246d)

**Comment by shiragoodman** [comment]
@oddball-lindsay they appear to be screenshots, not links. Looks like they're just not loading for you. Can you see the images below? If not, lmk and I can share in Slack.
<img width="604" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/71330581/1546a760-7a1a-480b-b7a4-07ca216c7966">
